lib/mayaUsd/ufe/UsdAttributes.cpp

```diff
@@ -112,6 +112,11 @@ static PXR_NS::UsdAttribute _GetAttributeType(const PXR_NS::UsdPrim& prim, const
 
 Ufe::Attribute::Type UsdAttributes::attributeType(const std::string& name)
 {
+    // early return if name is empty.
```

Comment:
Error handling. Required because Python code from Ufe checks the type before creating an attribute.